nixpkgs_review/nix.py

```diff
@@ -49,7 +50,8 @@ def nix_shell(
 ) -> None:
     nix_shell = shutil.which("nix-shell")
     if not nix_shell:
-        raise RuntimeError("nix-shell not found in PATH")
+        warn("nix-shell not found in PATH. Do you have nix installed?")
+        sys.exit(1)
```

Comment:
Actually we prefer `RuntimeError` or some other Error here, because raising an error will also call our cleanup code i.e. removing directories. You can add a new error type to catch if you want to make the generated error message nicer.

Reply:
Done.

I will leave to just raising an Exception them.